### DISCUSSION TITLE
[RFC Call] Agenda - March 1, 2021

### DISCUSSION THREAD

**Discussion by FredKSchott** [withastro/roadmap#129] [Announcements]
# To Discuss

- None yet

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)._

**Comment by natemoo-re** [comment]
I'd love to talk about exposing an `Astro.slots.render` util. https://github.com/withastro/astro/pull/2695
